### ISSUE TITLE
Document authorship policies

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [NOAA-FIMS/collaborative_workflow#152]
From Ian, Bai, Christine, Howard Townsend, and Patrick Lynch

1. We should refine the R package list in the DESCRIPTION file following R package standards to focus only on folks who have contributed directly to the package. No clear conclusion on how to separate "contributor" vs "author". Edit: forgot to add that we agreed @kellijohnson-NOAA should be "maintainer" to reflect status as acting FIMS lead.
2. We should create a free-form webpage similar to https://www.r-project.org/contributors.html which has a complete list of past members and advisors, contributors, etc. with more flexibility to describe the different roles.
3. Papers: the lead author has flexibility to set authorship based on who contributed to the writing and what the subject and audience are. In the case of the first paper, Christine will revise the author list based on this discussion.
4. Presentations: lead presenter has flexibility to list authors as they wish, title slide can include short list, perhaps including "FIMS Implementation Team". We can refine/improve the default slide listing the larger implementation team.

@kellijohnson-NOAA, feel free to suggest changes to these proposals, share them with the larger group, and/or task the Documentation team with getting to work on any of them (@Bai-Li-NOAA volunteered us with my support).

Additional notes from the discussion:

- R package order
  - Follow the guidelines from CRAN
  - How do we honor contributions to code review?
    - These could also count as contributions/authors depending on level of input
  - Generate a DOI for FIMS but update author list first
- Presentations
  - Depends on topic of presentation?
  - We did use the email list as an author for a recent presentation, and in general can list “FIMS Implementation Team”
- Manuscripts
  - Could have multiple tiers of authorship with alphabetical within each tier
  - Can we use the email list as an author?
  - Last author is sometimes the funder, but no clear standard in fisheries journals
- Some way of including a freeform contributors doc we can reference
  - Easier to list past contributors, folks who provide ideas but not code, etc.

_Originally posted by @iantaylor-NOAA in https://github.com/NOAA-FIMS/seaside-chats/issues/8#issuecomment-2251208247_

**Comment by kellijohnson-NOAA** [comment]
This task is on hold until we have a permanent FIMS Project Lead so we do not have to do things multiple times but should be done before the next release.
